docs/src/main/sphinx/connector/druid.rst

```diff
@@ -41,3 +41,19 @@ secured by basic authentication by updating the URL and adding credentials:
 
 Now you can access your Druid database in Trino with the ``druiddb`` catalog
 name from the properties file.
+
+.. _druid-sql-support:
+
+SQL support
+-----------
+
+The connector provides read access and write access to data and metadata in the
+Druid database. In addition to the :ref:`globally available
+<sql-globally-available>` and :ref:`read operation <sql-read-operations>`
+statements, the connector supports the following features:
+
+* :doc:`/sql/insert`
+* :doc:`/sql/delete`
+* :ref:`sql-schema-table-management`
```

Comment:
None of these are supported.

Reply:
Does that mean only SHOW statements are supported?

Reply:
Yeah, Druid is pretty much read-only.

Reply:
So the entire "in-addition to" part doesn't apply to Druid.

Reply:
What about the statements I've included in the file?